10_Sidebar/readme.md

```diff
@@ -134,6 +134,12 @@ _./src/index.html_
 +  <body id="main">
 ```
 
+_./src/main.tsx_
+
+```diff
++ document.getElementById('main')
+```
+
```

Comment:
This `main` and the change above (`<body id="main">`) has no sense and should be removed from readme. It also was not implemented in code. Could you remove it please?

Apply the same change on ES readme 🙏